sensor/sensor-model.go

```diff
@@ -21,7 +21,7 @@ type SensorRange struct {
 }
 
 type SensorRangeChange struct {
-	Sensor string  `json:"sensor"`
+	Sensor string  `json:"sensorType"`
```

Comment:
Are you sure we want to rename this field? In the database it is SENSOR as well and I thought, a shorter name is better.

Reply:
Then I must have misread the api.yaml. I will change it back.